Client/src/main/java/pl/jakubowskiprzemyslaw/tajgertim/controllers/ResultController.java

```diff
@@ -0,0 +1,35 @@
+package pl.jakubowskiprzemyslaw.tajgertim.controllers;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ResponseBody;
+import pl.jakubowskiprzemyslaw.tajgertim.models.Player;
+import pl.jakubowskiprzemyslaw.tajgertim.services.QueueService;
+import pl.jakubowskiprzemyslaw.tajgertim.services.SessionService;
+
+import javax.servlet.http.HttpServletRequest;
+import java.io.IOException;
+import java.util.concurrent.TimeoutException;
+
+@Controller
+public class ResultController {
```

Comment:
This name of class suggest that is Game Result of some Player

Reply:
Done